relayer/pkg/starknet/utils.go

```diff
@@ -87,26 +87,14 @@ func DecodeFelts(felts []*big.Int) ([]byte, error) {
 	return data, nil
 }
 
-// SignedBigToFelt wraps negative values correctly into felt
```

Comment:
No need for SignedBigToFelt anymore because there is no need to wrap around to a felt to represent negative numbers on-chain. In all places where this was used we can just use the big.Int value as is

Reply:
Question: do we want to keep SignedBigToFelt logic around though because it will also wrap values that are too large? Or should we error that values are beyond the capacity of felt252

Reply:
@archseer

Reply:
The wrapping was done because that's how negative values worked on 0.x, the upper part of the felt was treated as negative by certain functions. We should error here because this is not supported anymore and wrapping would produce invalid values (very large positive values)